src/org/daisy/dotify/formatter/impl/obfl/ObflParserImpl.java

```diff
@@ -1164,7 +1164,11 @@ private void parseTableOfContents(XMLEvent event, XMLEventIterator input, TextPr
 			if (equalsStart(event, ObflQName.TOC_BLOCK)) {
 				parseTocBlock(event, input, toc, tp);
 			} else if (equalsStart(event, ObflQName.TOC_ENTRY)) {
+                // this will throw a runtime exception, because a toc-entry must reside inside a toc-block
```

Comment:
Will it always throw an exception then that should be represented in the statement. Otherwise, it could throw a runtime exception if it's not in a toc-block.

Reply:
I have simply removed these two `if` statements.